oscd.go

```diff
@@ -27,23 +29,44 @@ func main() {
 	}
 
 	targetDir := os.Args[1]
+
+	// get abs path if given relative
+	if !filepath.IsAbs(targetDir) {
+		td, err := filepath.Abs(targetDir)
+		if err != nil {
+			panic(err)
+		}
+		targetDir = td
+	}
+
 	fmt.Printf("Target Directory: %s\n", targetDir)
 
 	// second arg onward is command to run
 	command := os.Args[2]
 	fmt.Printf("Command: %s\n", command)
 
+	// check if binary exists
```

Comment:
This is a little cleaner and a chance to show a better error. I just let the shell handle it 👍